src/main/java/com/scalar/cassy/util/RemoteFileSystemConnection.java

```diff
@@ -0,0 +1,43 @@
+package com.scalar.cassy.util;
+
+import com.google.common.base.Preconditions;
+import com.palantir.giraffe.host.Host;
+import com.palantir.giraffe.host.HostControlSystem;
+import com.palantir.giraffe.ssh.PublicKeySshCredential;
+import com.palantir.giraffe.ssh.SshCredential;
+import com.palantir.giraffe.ssh.SshHostAccessor;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Paths;
+
+public class RemoteFileSystemConnection {
```

Comment:
The `GiraffeUtil` class was refactored to contain an `SshHostAccessor` instance and the remote file system storage path.